libcalico-go/lib/epstatusfile/status_file_watcher_test.go

```diff
@@ -125,6 +153,9 @@ var _ = Describe("Workload endpoint status file watcher test", func() {
 
 		filePath := filepath.Join(statusDir, "pod1")
 		Eventually(haveEvents, "5s", "1s").WithArguments(filePath, []string{"create"}).Should(BeTrue())
```

Comment:
These assertions look race-prone to me.  haveEvents is reading data that's modified by callback but it doesn't use a mutex